### PR TITLE
FT: ZNC-26: add issue template

### PR DESCRIPTION
**Pull request by LaureVergeron**

### CODE REVIEW

**Review by smaffulli — COMMENTED** [review]
LGTM

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+### Usecase
+
+Please provide usecases for changing the current behavior
+
+### Additional information
+
+- Is this request for your company? Y/N
+
+..- If Y: Company name:
```
Comment (rahulreddy) [inline .github/ISSUE_TEMPLATE.md]: Is this expected to be a sub bullet? If yes, then adding two spaces before the `-` will make it one.

**Review by rahulreddy — COMMENTED** [review]
Thanks for the template 👍

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (LaureVergeron) [reply .github/ISSUE_TEMPLATE.md]: Yes, good catch - I fixed it on other repos but forgot utapi, thanks for the reminder!

**Pull request merged**